MultiVersions/Forge/main/meadow/lang/zh_tw.json

```
@@ -0,0 +1,249 @@
+  "advancement.meadow.straw_bed": "鄉村風小睡",
+  "advancement.meadow.straw_bed.desc": "在稻草床上忍耐一晚，醒來時背部酸痛",
+  "advancement.meadow.wool_collector": "動物毛愛好者",
+  "advancement.meadow.wool_collector.desc": "收集所有 7 種由《Meadow》模組所引入的獨特動物毛",
+  "block.meadow.alpine_birch_leaves_hanging": "懸掛的樺木樹葉",
+  "block.meadow.alpine_birch_log": "高山樺木原木",
+  "block.meadow.alpine_coal_ore": "高山煤礦",
+  "block.meadow.alpine_copper_ore": "高山銅礦",
+  "block.meadow.alpine_diamond_ore": "高山鑽石礦",
+  "block.meadow.alpine_emerald_ore": "高山綠寶石礦",
+  "block.meadow.alpine_gold_ore": "高山金礦",
+  "block.meadow.alpine_iron_ore": "高山鐵礦",
+  "block.meadow.alpine_lapis_ore": "高山青金石礦",
+  "block.meadow.alpine_poppy": "高山罌粟",
+  "block.meadow.alpine_redstone_ore": "高山紅石礦",
+  "block.meadow.alpine_salt_ore": "高山鹽礦",
+  "block.meadow.amethyst_cheese_block": "紫水晶乳酪輪",
+  "block.meadow.bench": "松木長椅",
+  "block.meadow.buffalo_cheese_block": "水牛乳酪輪",
+  "block.meadow.camera": "照相機",
+  "block.meadow.can": "牛奶桶",
+  "block.meadow.chair": "松木椅",
+  "block.meadow.cheese_block": "乳酪輪",
+  "block.meadow.cheese_form": "乳酪壓榨機",
+  "block.meadow.cheese_rack": "乳酪架",
+  "block.meadow.cheese_tart": "乳酪塔",
+  "block.meadow.cheesecake": "乳酪蛋糕",
+  "block.meadow.chiseled_limestone_bricks": "浮雕石灰岩",
+  "block.meadow.climbing_rope": "繩索",
+  "block.meadow.climbing_rope_topmount": "攀岩繩",
+  "block.meadow.cobbled_limestone": "石灰岩碎石",
+  "block.meadow.cobbled_limestone_slab": "石灰岩碎石半磚",
+  "block.meadow.cobbled_limestone_stairs": "石灰岩碎石階梯",
+  "block.meadow.cobbled_limestone_wall": "石灰岩碎石牆",
+  "block.meadow.cooking_cauldron": "料理鐵鍋",
+  "block.meadow.cracked_limestone_bricks": "裂紋石灰岩磚",
+  "block.meadow.delphinium": "翠雀花",
+  "block.meadow.doormat": "門墊",
+  "block.meadow.enzian": "龍膽",
+  "block.meadow.eriophorum": "羊鬍子草",
+  "block.meadow.eriophorum_tall": "高羊鬍子草",
+  "block.meadow.fire_lily": "珠芽百合",
+  "block.meadow.fire_log": "柴火堆",
+  "block.meadow.flecked_bed": "斑點羊毛床",
+  "block.meadow.flecked_carpet": "斑點羊毛地毯",
+  "block.meadow.flecked_wool": "斑點羊毛",
+  "block.meadow.fondue": "乳酪火鍋",
+  "block.meadow.frame": "營火架",
+  "block.meadow.goat_cheese_block": "山羊乳酪輪",
+  "block.meadow.grain_cheese_block": "榖物乳酪輪",
+  "block.meadow.heart_patterned_window": "愛心花紋窗戶",
+  "block.meadow.highland_bed": "高地牛毛床",
+  "block.meadow.highland_carpet": "高地牛毛地毯",
+  "block.meadow.highland_wool": "高地牛毛",
+  "block.meadow.inky_bed": "墨水羊毛床",
+  "block.meadow.inky_carpet": "墨水羊毛地毯",
+  "block.meadow.inky_wool": "墨水羊毛",
+  "block.meadow.limestone": "石灰岩",
+  "block.meadow.limestone_brick_slab": "石灰岩磚半磚",
+  "block.meadow.limestone_brick_stairs": "石灰岩磚階梯",
+  "block.meadow.limestone_brick_wall": "石灰岩磚牆",
+  "block.meadow.limestone_bricks": "石灰岩磚",
+  "block.meadow.limestone_slab": "石灰岩半磚",
+  "block.meadow.limestone_stairs": "石灰岩階梯",
+  "block.meadow.limestone_wall": "石灰岩牆",
+  "block.meadow.meadow.alpine_salt": "高山鹽",
+  "block.meadow.meadow_banner": "完美收集旗幟：§2Meadow",
```

原版旗幟不是有翻譯嗎，還是這是不一樣的東西
```suggestion
  "block.meadow.meadow_banner": "完美收集旗幟：§2Meadow",
```

---

不一樣的東西